CHANGELOG.md

```diff
@@ -1,21 +1,9 @@
-# NEXT RELEASE
-
-### Enhancements
-* None.
+# 11.4.1 Release notes
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
 * Fixed race between updating user tokens and state in persisted metadata when logging in/out. ([#4875](https://github.com/realm/realm-core/issues/4875))
```

Comment:
How could an end user experience this?

Reply:
I noticed a mention of this issue in a user report that had this crash - is this fixing that? Then that could be written:
> /Users/realm/workspace/realm_realm-js_master/vendor/realm-core/src/realm/object-store/sync/sync_user.cpp:281: [realm-core-11.0.4] Assertion failed: m_state == SyncUser::State::LoggedIn

Reply:
No idea. I did notice that the end user experience was missing, but I could not figure out what the symptom could be.

Reply:
Fixed